turbinia/config/turbinia_config_tmpl.py

```diff
@@ -73,6 +73,34 @@
 # problems.
 DEBUG_TASKS = False
 
+################################################################################
+#                         External Dependency Configurations
+#
+# Options in this section are used to configure system and docker dependencies.
+################################################################################
+
+# This will allow for the configuration of system dependencies and docker
+# containers. The following configuration will need to be set per job config.
+# Please use the example below as a guidance to adding additional job checks.
+# {
+#   'job': 'The name of the job.'
+#   'programs': 'List of programs to check for.
+#   'docker_image': name of docker image (if any).
```

Comment:
Another minor nit: The quoting of the value strings is different for all three of these lines :).  Maybe have something like:
```
# {
#   'job': 'MyJob'
#   'programs': ['list', 'of', 'required', 'programs']
#   'docker_image': 'Name of docker image'  # Can also be set to None if there is not an image for this Job.
# }
```
My docker_image example string could definitely be better, but I wanted to include somewhere that it could be set to `None`.